torchdynamo/variables/tensor.py

```diff
@@ -366,12 +367,14 @@ class TensorWithTFOverrideVariable(VariableTracker):
     def __init__(
         self,
         tensor_variable,
+        orig_tensor_variable_source,
         subclass_torch_function__func,
         subclass_type,
         **kwargs,
     ):
         super(TensorWithTFOverrideVariable, self).__init__(**kwargs)
         self.tensor_variable = tensor_variable
+        self.orig_tensor_variable_source = orig_tensor_variable_source
```

Comment:
Should this go in the `self.source` field (defined in the VariableTracker base class)?

The advantage of that is it will work with `resume_at_` continuations.

Reply:
I actually tried that first, that led to problems in this line: https://github.com/pytorch/torchdynamo/blob/main/torchdynamo/codegen.py#L88, that statement would now pass for `TensorWithTFOverrideVariable` and it would lead to incorrect graph outputs.

We could unbreak it by doing something like

```
        if value.source is not None and allow_cache and not isinstance(value, TensorWithTFOverrideVariable):
            output.extend(value.source.reconstruct(self))
```

but that also looked pretty hacky.  WDYT?

Reply:
Ah got it, I'm fine with this version then.